src/tests/unit/test-util.js

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
This file isn't used anywhere. I'm assuming they are for the serializer tests which are also missing.